vk/server.go

```diff
@@ -28,14 +29,15 @@ type Server struct {
 func New(opts ...OptionsModifier) *Server {
 	options := newOptsWithModifiers(opts...)
 
-	router := NewRouter(options.Logger)
-	router.useQuietRoutes(options.QuietRoutes)
+	internalRouter := NewRouter(options.Logger)
+	internalRouter.useQuietRoutes(options.QuietRoutes)
 
 	s := &Server{
-		router:  router,
-		lock:    sync.RWMutex{},
-		started: atomic.Value{},
-		options: options,
+		router:         options.RouterWrapper(internalRouter),
```

Comment:
Don't think we need to set this here, since it gets set up in `Start` ?

Reply:
Good spot, thank you! Removed